### ISSUE TITLE
".nuxt/paths.mjs" error when creating a 3.3.1 project

### ISSUE THREAD

**Issue by Posoroko** [nuxt/nuxt#19734]
### Environment

npx nuxi info:

- Operating System: `Windows_NT`
- Node Version:     `v18.15.0`
- Nuxt Version:     `3.3.1`
- Nitro Version:    `2.3.1-27979121.b167c62`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

https://github.com/Posoroko/Krismen2.0

I created this 3.3.1 project and the error comes when running "npm run dev"

### Describe the bug

when running npm run dev, I get this error:

 ERROR  Failed to load url /@fs/D:/Users/<windows user>/Documents/projects/node_modules/ufo/dist/index.mjs?v=bfaf0afc (resolved id: D:/Users/<windows user>/Documents/projects/node_modules/ufo/dist/index.mjs?v=bfaf0afc) in virtual:nuxt:D:/Users/<windows user>/Documents/projects/Krismenn/Krismenn2.0/.nuxt/paths.mjs. Does the file exist?

This error goes away if I manually create the paths.mjs file inside the .nuxt folder.

**Comment by danielroe** [comment]
I can't reproduce this. What package manager are you using? (Your project has both yarn + npm lockfiles.) Maybe try deleting those lockfiles, clearing out `node_modules` and reinstalling?

**Comment by danielroe** [comment]
You could also try removing your `.nuxt` folder. And note that it seems to be resolving from `Users//Documents/projects/node_modules` when your project is actually somewhere else. Is there a reason for that? You can try deleting that extra node_modules folder.

**Comment by Posoroko** [comment]
Yeah I just realised this. I think I got mixed up and didn't realize about the other node_modules folder. I looks like all my projects were using this same folder (if it's possible)  so when I upgraded one of them, all the other ones got the same error.
(I might have done npm install before cd'ing into the folder. Could that be it? 

anyways... I'm sorry for that. It's totally a noob error. :-)  Thanks for you time and help!
